### PR TITLE
Test

### PR DESCRIPTION
**Pull request by KarinLee**
Add script to run test cases, generate test log file, and update App.config property value with command line parameter.
Update playlist according to remove non-exist test cases.

### CODE REVIEW

File: DevOfficeComTest/Script/runTest.cmd [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ echo.
+ echo Options:
+ echo.
+ echo [/Browser:^<Browser Name^>]
+ echo The browser used to access the site. The value should be one of following: IE32, IE64, Chrome, Firefox.
+ echo.
+ echo [/BaseAddress:^<Base Url^>]
+ echo The root url of the site under test.
+ echo.
+ echo [/ScreenShotSavePath:^<Save Path^>]
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: can we comment out this config property for now? as we are not using it, people may confused.

File: DevOfficeComTest/Script/runTest.cmd [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+ echo An absolute or relative path to save the screenshot file.
+ echo.
+ echo [/DefaultWaitTime:^<Wait Time^>]
+ echo Default wait time when finding element.
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: add somehting like: currently default value is 15 seconds, if you don't change to change, ignore this parameter.

File: DevOfficeComTest/Script/runTest.cmd [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+ echo.
+ echo [/Tests:^<Test Case Name[,Test Case Name]...^>]
+ echo Run tests with names that match the provided values.
+ echo More details see vstest.console.exe /?
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: write something for if ignore this, will run all by default?

File: DevOfficeComTest/Script/runTest.cmd [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+ echo.
+ echo [/TestCaseFilter:^<Expression^>]
+ echo Run tests that match the given expression.
+ echo ^<Expression^> is of the format ^<property^>Operator^<value^>[^|^&^<Expression^>]
+ echo More details see vstest.console.exe /?
+ echo.
+ echo [/PlayList:^<PlayList File Name^>]
+ echo A playList file which contains a list of tests to be run.
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: write something for if ignore this, will run all by default?

File: DevOfficeComTest/Script/runTest.cmd [hunk 5]
```diff
@@ -0,0 +1,147 @@
+@ECHO OFF
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: do we need another cmd for graph site as well?

File: DevOfficeComTest/Script/runTest.cmd [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+ echo More details see vstest.console.exe /?
+ echo.
+ echo [/TestCaseFilter:^<Expression^>]
+ echo Run tests that match the given expression.
+ echo ^<Expression^> is of the format ^<property^>Operator^<value^>[^|^&^<Expression^>]
+ echo More details see vstest.console.exe /?
+ echo.
+ echo [/PlayList:^<PlayList File Name^>]
+ echo A playList file which contains a list of tests to be run.
+ goto end
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: let's give an example cmd in help

File: DevOfficeComTest/Script/runTest.cmd [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+  for %%d in (%%c) do (
+   set te=%%d
+   if "!te:~1,5!"=="Tests" (
+    set tests=!tests!!te:~1,-1!,
+   )
+  )
+ )
+ vstest.console.exe .\Tests\bin\Debug\Tests.dll /logger:trx !tests:~0,-1!
+) else (
+ vstest.console.exe .\Tests\bin\Debug\Tests.dll /logger:trx
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: can we make the log file path as a parameter as well?

File: DevOfficeComTest/Script/runTest.cmd [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+ set testFilter=!te!
+ )
+if "!flag3!"=="1" (
+ if not "!te:~0,1!"=="/" (
+   set testCases=!testCases!,!te!
+  )
+ )
+)
+if defined testFilter (
+ vstest.console.exe .\Tests\bin\Debug\Tests.dll /logger:trx !testFilter!
```
Comment (JingyuShao) [inline DevOfficeComTest/Script/runTest.cmd]: can we add first step as build the solution ?

File: DevOfficeComTest/Script/runTest.cmd (hunk 1 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: Ok.

File: DevOfficeComTest/Script/runTest.cmd (hunk 2 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: Accept.

File: DevOfficeComTest/Script/runTest.cmd (hunk 3 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: Accept. All tests will be run if no parameter provided.

File: DevOfficeComTest/Script/runTest.cmd (hunk 4 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: Accept.

File: DevOfficeComTest/Script/runTest.cmd (hunk 5 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: I think if we add a parameter [TestFileNames] maybe this cmd can also be used for graph site. I will try this today.

File: DevOfficeComTest/Script/runTest.cmd (hunk 6 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: Accept.

File: DevOfficeComTest/Script/runTest.cmd (hunk 8 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: I will try this today.

File: DevOfficeComTest/Script/runTest.cmd (hunk 5 above)
Comment (KarinLee) [inline DevOfficeComTest/Script/runTest.cmd]: As we have discussed, I will add a cmd for graph site as well.

File: MSGraphTest/TestFramework/App.config [hunk 9]
```diff
@@ -21,6 +24,6 @@
     <!--The user name for MS Graph explorer login-->
     <add key="GraphExplorerUserName" value="admin@arvindsa.onmicrosoft.com"/>
     <!--The password for MS Graph explorer login-->
-    <add key="GraphExplorerPassword" value="DevExRocks!3"/>
+    <add key="GraphExplorerPassword" value="DevExRocks3"/>
```
Comment (JingyuShao) [inline MSGraphTest/TestFramework/App.config]: why change the password? I think it's still with !

Comment (KarinLee) [inline MSGraphTest/TestFramework/App.config]: Hi Jingyu,

Sorry, the password maybe changed when I update App.config of MSGraph by mistake.

**Pull request merged**